controllers/csm_controller.go

```diff
@@ -1347,7 +1347,8 @@ func (r *ContainerStorageModuleReconciler) PreChecks(ctx context.Context, cr *cs
 	if err != nil {
 		return fmt.Errorf("failed upgrade check: %v", err)
 	} else if !upgradeValid {
-		return fmt.Errorf("failed upgrade check because upgrade is not valid")
+		log.Infof("upgrade is not valid")
```

Comment:
no linting error here? PowerScale driver this was throwing error. anything with ending "f" need format.

Reply:
No this wasn't flagged for linting.